server/routes/integrations/__tests__/integrations_router.test.ts

```diff
@@ -0,0 +1,46 @@
+import { DeepPartial } from 'redux';
+import { OpenSearchDashboardsResponseFactory } from '../../../../../../src/core/server/http/router';
+import { handleWithCallback } from '../integrations_router';
+import { IntegrationsAdaptor } from 'server/adaptors/integrations/integrations_adaptor';
+
+jest
+  .mock('../../../../../../src/core/server', () => jest.fn())
```

Comment:
np. there is a public corestart mock, not sure if there is a corresponding one for server that can be used https://github.com/opensearch-project/dashboards-observability/blob/ebecc31f451c547dab9dff3489da409669c86ddb/test/__mocks__/coreMocks.ts#L11